### PR TITLE
bump node image to 12 in circleci 

### PR DESCRIPTION
**Pull request by csjoblom**

### CODE REVIEW

**Comment by csjoblom** [comment]
blocked by #106 tests should pass once we have that merged in.

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:9.8.0-browsers
+      - image: circleci/node:11
```
Comment (ryanrolds) [inline .circleci/config.yml]: Odd versions of Node are non-LTS releases. Node.js 10.x or Node.js 12.x is what should be used.

Comment (ryanrolds) [reply .circleci/config.yml]: https://nodejs.org/en/about/releases/

Comment (ryanrolds) [reply .circleci/config.yml]: Node.js 11 isn't even supported currently.

**Comment by csjoblom** [comment]
I’ll bump to 12 and update my other PR

File: .circleci/config.yml (hunk 1 above)
Comment (csjoblom) [reply .circleci/config.yml]: I bumped it up to 12

**Comment by csjoblom** [comment]
Closing this and will add to #106 instead.

**Pull request closed without merging**